ereuse_devicehub/resources/documents/templates/documents/erasure.html

```diff
@@ -42,13 +42,13 @@ <h4>{{ erasure.device.__format__('t') }}</h4>
 
                     <dt>Computer where was erase:</dt>
 		    <dd>Title: {{ erasure.parent.__format__('ts') }}</dd>
-		    <dd>SystemId: {{ erasure.parent.id }}</dd>
+		    <dd>DevicehubID: {{ erasure.parent.devicehub_id }}</dd>
 		    <dd>Hid: {{ erasure.parent.hid }}</dd>
 		    <dd>Tags: {{ erasure.parent.tags }}</dd>
 		    
                     <dt>Computer where it resides:</dt>
 		    <dd>Title: {{ erasure.device.parent.__format__('ts') }}</dd>
-		    <dd>SystemId: {{ erasure.device.parent.id }}</dd>
+		    <dd>DevicehubID: {{ erasure.device.parent.devicehub_id }}</dd>
 		    <dd>Hid: {{ erasure.device.parent.hid }}</dd>
 		    <dd>Tags: {{ erasure.device.parent.tags }}</dd>
 
```

Comment:
Line 90: `<a href="{{ url_web }}">Verify on-line the integrity of this document</a>`

Update _url_web_  variable (public link) with DevicehubID instead of SystemID?

Reply:
changed for stamp url